mlir/tools/rocmlir-lib/CMakeLists.txt

```diff
@@ -78,9 +79,10 @@ if(BUILD_FAT_LIBROCKCOMPILER)
     rocm_install(FILES ${__mlir_libraries}
             DESTINATION lib/llvm)
 
-    rocm_install(FILES ${ROCMLIR_LIB_DIR}/MLIRRockThin.lib
+    rocm_install(FILES ${__rocmlir_libraries}
```

Comment:
Then shouldn't this also include `${__mlir_libraries}`

Reply:
The line above does `${__mlir_libraries}` separately and install the libraries into the `lib\llvm` directory. The rocMLIR libraries are installed into the `lib` directory. That is the safety precaution for rocMLIR to interface with the correct libraries (other components we depend on are using llvm-project in different versions and scopes).
Of course, there is no issue when someone installs into `<base>/rocmlir/`. However, we cannot guarantee following that rule. Instead, one may install it in a shared location overriding/skipping existing files and creating hard-to-find bugs.

Reply:
Makes sense